testsuite/tests/lib-runtime-events/test_create_cursor_failures.ml

```diff
@@ -2,6 +2,7 @@
  include unix;
  include runtime_events;
  hasunix;
+ not-beos_haiku;
```

Comment:
I've also hit a failure here when testing https://github.com/ocaml/ocaml/pull/14431, however i think it would be more elegantly fixed by adding a way to detect if the current user is a super-user.

The issue is that the test does `chmod 000 <some file>` expecting the file to be unreadable but `root` on any Unix compatible system will simply read it regardless.

Reply:
https://github.com/ocaml/ocaml/pull/14435 has now been merged so this and its definition(s) can now be removed
```suggestion
```